bgpd/bgp_evpn.c

```diff
@@ -3714,9 +3712,7 @@ static int delete_withdraw_vni_routes(struct bgp *bgp, struct bgpevpn *vpn)
 	/* Delete and withdraw locally learnt type-2 routes (MACIP)
 	 * for this VNI - from the global table.
 	 */
-	ret = delete_global_type2_routes(bgp, vpn);
-	if (ret)
-		return ret;
+	delete_global_type2_routes(bgp, vpn);
```

Comment:
Let's convert this function to void then too, it's absolutely useless keeping it as int, it returns nothing.

Reply:
Done, thanks.